workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowDefinitionViewerPage/WorkflowDefinitionViewerPage.tsx

```diff
@@ -14,113 +14,50 @@
  * limitations under the License.
  */
 import React, { useMemo } from 'react';
-import { useNavigate } from 'react-router-dom';
-import { useAsync } from 'react-use';
 
-import { InfoCard, ResponseErrorPanel } from '@backstage/core-components';
-import {
-  useApi,
-  useRouteRef,
-  useRouteRefParams,
-} from '@backstage/core-plugin-api';
+import { InfoCard } from '@backstage/core-components';
+import { useRouteRefParams } from '@backstage/core-plugin-api';
 
-import { Button, Grid, Tooltip } from '@material-ui/core';
-import { Skeleton } from '@material-ui/lab';
+import { Grid } from '@material-ui/core';
 
-import {
-  orchestratorWorkflowUsePermission,
-  orchestratorWorkflowUseSpecificPermission,
-} from '@red-hat-developer-hub/backstage-plugin-orchestrator-common';
-
-import { orchestratorApiRef } from '../../api';
-import { usePermissionArrayDecision } from '../../hooks/usePermissionArray';
-import {
-  executeWorkflowRouteRef,
-  workflowDefinitionsRouteRef,
-} from '../../routes';
-import { BaseOrchestratorPage } from '../BaseOrchestratorPage';
+import { workflowRouteRef } from '../../routes';
 import { EditorViewKind, WorkflowEditor } from '../WorkflowEditor';
 import WorkflowDefinitionDetailsCard from './WorkflowDefinitionDetailsCard';
 
-export const WorkflowDefinitionViewerPage = () => {
```

Comment:
I suggest to rename the component and file to WorkflowDetailsTabContent, since Page indicates that it's a whole page

Reply:
should I change the name of the matching file as well?
what about the card name?
![image](https://github.com/user-attachments/assets/895d7e75-61f8-49af-a085-7dc569de45ac)

Reply:
yes, also the the matching file
The card name is fine